A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_PerformancePage.js

```diff
@@ -588,9 +588,17 @@ class CDUPerformancePage {
         if (isFinite(mcdu.perfApprDH)) {
             dhCell = mcdu.perfApprDH.toFixed(0);
         }
+        if(mcdu.perfApprDH == -1) {
+            dhCell = "NO";
+        }
         mcdu.onRightInput[2] = () => {
             let value = mcdu.inOut;
             mcdu.clearUserInput();
+            if(value == 'NO' || 'no') {
+                mcdu.setPerfApprDH(-1);
+                CDUPerformancePage.ShowAPPRPage(mcdu);
+                mcdu.clearUserInput();
```

Comment:
**question**: Do any of the function calls above this one set input?
**suggestion**: If not, then remove this call to `clearUserInput`, because it was already cleared earlier on line 596.